### PR TITLE
Fix typo & add hbs to code fences where missing

### PR DESCRIPTION
**Pull request by zeppelin**

### CODE REVIEW

**Comment by zeppelin** [comment]
Btw I :heart: the proposal!

File: active/0000-improved-actions.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ Lastly, closure actions allow for yielding an action to a block. For example:
 
 ```hbs
 {{! app/index/template.hbs }}
-{{my-form save=(action 'submit') as |submit reset}}}
+{{my-form save=(action 'submit') as |submit reset|}}}
```
Comment (hibariya) [inline active/0000-improved-actions.md]: extra `}`?

Comment (rwjblue) [inline active/0000-improved-actions.md]: Yes, should only be two closing }}.

Comment (zeppelin) [inline active/0000-improved-actions.md]: @rwjblue @hibariya sorry, updated!

**Comment by rwjblue** [comment]
Thanks!

**Pull request merged**